### PR TITLE
Daemonization Work

### PR DESCRIPTION
**Pull request by stinkerton18**
New ups.service file for daemonization, updated SETUP.md with updated commands to setup automated start on boot.

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -67,9 +67,10 @@ python3 ups.py
 ***This section is NOT complete!!!***
 
 ```
-sudo cp startup/ups.service /lib/systemd/system/
-sudo chmod 644 /lib/systemd/system/ups.service
+sudo cp SuperUPS/host_monitor/ups*.py /usr/bin/
```
Comment (akuker) [inline SETUP.md]: Could we put these in /usr/local/bin instead of /usr/bin?

File: SETUP.md [hunk 2]
```diff
@@ -67,9 +67,10 @@ python3 ups.py
 ***This section is NOT complete!!!***
```
Comment (akuker) [inline SETUP.md]: This section IS complete now :-)

File: host_monitor/ups.service [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[Unit]
```
Comment (akuker) [inline host_monitor/ups.service]: Can we rename file to "superups.service"?

**Review by akuker — CHANGES_REQUESTED** [review]

File: SETUP.md (hunk 1 above)
Comment (stinkerton18) [reply SETUP.md]: Can do, will update here shortly

File: SETUP.md (hunk 2 above)
Comment (stinkerton18) [reply SETUP.md]: True... :)

**Review by akuker — APPROVED** [review]

**Pull request merged**